plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/portforwarding/PortForwardingUtils.java

```diff
@@ -53,15 +53,11 @@ public static boolean isPortInUse(int port) {
 			return false;
 		}
 		try (ServerSocket socket = new ServerSocket(port)){
-			//not in use apparently
+			//not in use
+		    return false;
```

Comment:
@adietish , the previous logic was implemented when doing JBIDE-21847. It seemed then that the check with ServerSocket was not always sufficient. @fbricon , could you please try to remember why we kept both checks?